docs/user-manual.md

```diff
@@ -0,0 +1,471 @@
+# User manual
+
+The Bitfinex Report Tools allows traders to more conveniently track, manage and analyse trading/funding activity across their Bitfinex account. Built to be both faster and easier to use, traders can now instantly compile an overview of necessary account information. A highlight of this software is the fact that all data is synced locally, automatically. The sync experience has been developed in a manner which is as smooth and simple as possible, and which allows you to always remain up to date with account developments
+
+## Report sections
+
+The Tool provides the following main reports sections
+
+<details>
+
+<summary>Types of filters available</summary>
+
+> The columns might be filtered, so as to have a smaller screen, thereby removing information that is not required for analysis
+> Abailable the following types of filters:
+> - for string:
+>   - contains
+>   - begins with
+>   - ends with
+>   - equal to
+>   - not equal to
+> - for number
+>   - equal to
+>   - not equal to
+>   - greater than
+>   - greater/equal
+>   - less than
+>   - less/equal
+> - for date
+>   - before
+>   - equal to
+>   - not equal to
+> - for specific columns provides `equal to` and `not equal to` by selection list
+
+</details>
+
+<details>
+
+<summary>Ledgers</summary>
+
+> Shows your past ledger entries. Most recent entries are returned by default, but a timestamp can be used to retrieve time-specific data. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+
+</details>
+
+<details>
+
+<summary>Trades</summary>
+
+> - Shows your trades. Your most recent trades will be retrieved by default, but a timestamp can be used to retrieve time-specific data. Contain sections `Date Range Selection`, `Filters`, `Refresh Button`, `Table`, `Pagination`
+> - Shows candles, provides OCHL (Open, Close, High, Low) and volume data for the specified funding currency or trading pair. This section provides optional functionality to sync certain pairs and time period. Contain sections `Date Range Selection`, `Filters`, `Query Button`, `Refresh Button`, `Sync Preferences`, `Chart`
```

Comment:
Shows candles**, displays your trades and trade information over** the market trading candles providing OCHL (Open, Close, High, Low) and volume data for the specified funding currency or trading pair.... etc